api/lib/application/user-orga-settings/user-orga-settings-controller.js

```diff
@@ -16,5 +16,9 @@ module.exports = {
     const result = await usecases.createUserOrgaSettings({ userId, organizationId });
 
     return h.response(userOrgaSettingsSerializer.serialize(result)).created();
+  },
+
+  update() {
+    return null;
```

Comment:
?

Reply:
La méthode est implémentée dans un des commits suivants. Pour plus de clarté je vais remplacer ce "return null" par le commentaire suivant: "// not yet implemented"